src/DIPS.Xamarin.UI/Controls/Skeleton/SkeletonView.xaml.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace DIPS.Xamarin.UI.Controls.Skeleton
+{
+    /// <summary>
+    /// Skeleton view to show skeleton structure when data is loading.
+    /// </summary>
+    [ContentProperty(nameof(MainContent))]
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class SkeletonView : ContentView
+    {
+        private const string m_animationName = "SkeletonBounce";
+        private Grid m_skeletongrid;
+        private Grid? m_skeletonLayout;
+        private List<BoxView> m_skeletons = new List<BoxView>();
+        /// <summary>
+        /// Creates a new instance of skeleton view
+        /// </summary>
+        public SkeletonView()
+        {
+            Content = m_skeletongrid = new Grid();
```

Comment:
Add a preview feature flag and do a check if this is set. If it's not, you can throw an exception to let the user know that this is a preview feature and it need to be opted in :) 

This is to make it clear that you are using preview features